src/Umbraco.Community.Sustainability/Migrations/02_AddCarbonRating.cs

```diff
@@ -14,7 +14,7 @@ protected override void Migrate()
         {
             Logger.LogDebug("Running migration {MigrationStep}", "AddCarbonRating");
 
-            if (ColumnExists(PageMetric.TableName, "CarbonRating") == false)
+            if (ColumnExists(PageMetric.TableName, "CarbonRating"))
```

Comment:
This looks to flip the logic, so won't run the migration if the column doesn't exist.

Reply:
Reverted this change